src/sth_server.js

```diff
@@ -0,0 +1,151 @@
+/* globals module, process, require */
+
+(function() {
+  "use strict";
+
+  var hapi = require('hapi');
+  var joi = require('joi');
+
+  var server, sthDatabase, sthConfig, sthLogger, sthHelper;
+
+  function startServer(host, port, aSTHDatabase, callback) {
```

Comment:
JSDoc comments included! ;) Sorry about that, I forgot to do it from the beginning ;) I'll mention you as soon as I pushed the changes to the repo ;) Thanks!